code/addons/vitest/src/components/TestProviderRender.tsx

```diff
@@ -182,123 +181,123 @@ export const TestProviderRender: FC<TestProviderRenderProps> = ({
+              padding="none"
+              size="medium"
+              variant="ghost"
+              onClick={() =>
+                store.send({
+                  type: 'CANCEL_RUN',
+                })
+              }
+              disabled={cancelling || isStarting}
             >
-              <IconButton
-                aria-label={cancelling ? 'Stopping...' : 'Stop test run'}
-                padding="none"
-                size="medium"
-                onClick={() =>
-                  store.send({
-                    type: 'CANCEL_RUN',
-                  })
+              <Progress
+                percentage={
+                  finishedTestCount && storeState.currentRun.totalTestCount
+                    ? (finishedTestCount / storeState.currentRun.totalTestCount) * 100
+                    : undefined
                 }
-                disabled={cancelling || isStarting}
               >
-                <Progress
-                  percentage={
-                    finishedTestCount && storeState.currentRun.totalTestCount
-                      ? (finishedTestCount / storeState.currentRun.totalTestCount) * 100
-                      : undefined
-                  }
-                >
-                  <StopIcon />
-                </Progress>
-              </IconButton>
-            </WithTooltip>
+                <StopIcon />
+              </Progress>
+            </Button>
           ) : (
-            <WithTooltip
-              hasChrome={false}
-              trigger="hover"
-              tooltip={<TooltipNote note="Start test run" />}
+            <Button
+              ariaLabel="Start test run"
+              padding="small"
+              size="medium"
+              variant="ghost"
+              onClick={() =>
+                store.send({
+                  type: 'TRIGGER_RUN',
+                  payload: {
+                    storyIds: entry ? api.findAllLeafStoryIds(entry.id) : undefined,
+                    triggeredBy: entry ? entry.type : 'global',
+                  },
+                })
+              }
             >
-              <IconButton
-                aria-label="Start test run"
-                size="medium"
-                onClick={() =>
-                  store.send({
-                    type: 'TRIGGER_RUN',
-                    payload: {
-                      storyIds: entry ? api.findAllLeafStoryIds(entry.id) : undefined,
-                      triggeredBy: entry ? entry.type : 'global',
-                    },
-                  })
-                }
-              >
-                <PlayHollowIcon />
-              </IconButton>
-            </WithTooltip>
+              <PlayHollowIcon />
+            </Button>
           )}
         </Actions>
       </Heading>
-
+      {/* FIXME: aria labels and tooltips in this entire section need to be reviewed. We should state what we navigate to when clicking. */}
```

Comment:
This will need to be a separate ticket. We'll want to rethink the UI to see if we can separate status reporting from navigation a little bit more. Given the scale of work, it shouldn't be handled now.

* [ ] Remove comment
* [ ] Make issue to ensure a follow up, if none exists